containerd-shim-v2/service.go

```diff
@@ -665,6 +665,18 @@ func (s *service) Kill(ctx context.Context, r *taskAPI.KillRequest) (_ *ptypes.E
 		return nil, err
 	}
 
+	// If the container has stopped, return directly when killed with
+	// SIGKILL or SIGTERM signal.
```

Comment:
Why we still send other signal except SIGKILL and SIGTERM  when container is stop

Reply:
We cannot make sure when/what signals containerd would send, here only filter out the sigterm/sigkill signal.

Reply:
I think the signal should be send to container, if the container init process are stopped, it won't deal the signal, I see kata agent will discard signal if container stop, so it also ok here.
https://github.com/kata-containers/agent/blob/master/grpc.go#L908

Reply:
Actually if the container has been stopped, the signal hadn't reached agent and would returned here https://github.com/kata-containers/runtime/blob/a7e2bbd31c5689255abe618730fe7b1b017aaf48/virtcontainers/container.go#L1075.

Here the filter would make sure even the container has been stopped, send SIGKILL/SIGTERM  signal wouldn't return an err.